src/box/lua/tuple.c

```diff
@@ -757,6 +757,30 @@ lbox_tuple_info(lua_State *L)
 	return 1;
 }
 
+/**
+ * Push to Lua stack a table with the information about a format of a tuple.
+ * Elements of a table are structs {'name', 'type'}.
+ */
+static int
+lbox_tuple_get_format(lua_State *L)
+{
+	int argc = lua_gettop(L);
+	if (argc != 1)
+		luaL_error(L, "Usage: tuple:format()");
+
+	struct tuple *tuple = luaT_checktuple(L, 1);
+	struct tuple_format *format = tuple_format(tuple);
+	tuple_format_ref(format);
+
+	struct tuple_format **format_ptr =
+		lua_newuserdata(L, sizeof(*format_ptr));
+
+	luaL_getmetatable(L, "box.tuple.format");
```

Comment:
We need to create a constant for the string.
```suggestion
	luaL_getmetatable(L, "box.tuple.format");
```

Reply:
This line was removed, so no constants were added.